app/frontend/src/components/Map.tsx

```diff
@@ -46,13 +47,33 @@ export default function Map({
 
   const containerRef = useRef<HTMLDivElement>(null);
 
+  /*
+  Allows sending cookies (counted as sensitive "credentials") on cross-origin requests when we grab GeoJSON/other data from the API.
+
+  Those APIs will return an error if the session cookie is not set as these APIs are secure and not public.
+  */
+  const transformRequest = (
+    url: string,
+    resourceType: ResourceType
+  ): RequestParameters => {
+    if (url.startsWith(URL)) {
+      return {
+        url: url,
```

Comment:
You can use shorthand notation:
```suggestion
        url,
```